core/utils/dmljob.py

```diff
@@ -41,6 +41,9 @@ def __init__(
         self.hyperparams = hyperparams
         self.label_column_name = label_column_name
 
+    def add_weights(self, serialized_weights):
```

Comment:
I'm assuming `serialized_weights` is the output of calling `core.utils.keras.serialize_weights()` on a `job.weights`. Can you confirm?

What's the general purpose of this function and why does a DMLJob need to have both `.job` and `.serialized_weights`?

Reply:
- yes
- for averaging but i'll probably end up changing this in a subsequent PR for the Averaging Job

Reply:
Ok, will add a comment about that right now. We will let it slide to `master` momentarily.

Reply:
Let's remove this since it's not being used at all. Let's table it for the next PR.